src/R/Wpf/Impl/Microsoft.R.Wpf.csproj

```diff
@@ -0,0 +1,80 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We already have a Microsoft.Common.Wpf project, is this worth creating another project for?